### DISCUSSION TITLE
product detail image delay even install and use sharp (npm)

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [vercel/next.js#39756] [Help]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

Operating System:
      Platform: linux
      Arch: x64
      Version: #49-Ubuntu SMP Thu Aug 4 18:03:25 UTC 2022
    Binaries:
      Node: 18.7.0
      npm: 8.17.0
      Yarn: 1.22.19
      pnpm: 7.9.0
    Relevant packages:
      next: 12.2.6-canary.1
      eslint-config-next: 12.2.5
      react: 18.2.0
      react-dom: 18.2.0

### What browser are you using? (if relevant)

Brave Version 1.42.97 Chromium: 104.0.5112.102 (Official Build) (64-bit)

### How are you deploying your application? (if relevant)

next run build , next start

### Describe the Bug

I use SSG for dynamic page and image load very fast when it size is 100 width and height in next/Image.
Image is small and I resize width 400 and height . Then My image have some delay. I also use sharp and It still happening .

### Expected Behavior

I want big width height image without delay

### Link to reproduction

https://jherr-pokemon.s3.us-west-1.amazonaws.com/index.json

### To Reproduce

I fetch all data with fetch (useEffect) .
Create dyanmic pages and display the single pokemon with name and image.
I use next/image and sharp .
-> npm run build   ,   npm start
Set width and height to 100 and see result is image small and fast.
Now
Change width and height to 400 and see result image is big but have delay problem.

**Comment by awareness481** [comment]
If the image is above the fold (at the top of your page), you might want to use https://nextjs.org/docs/api-reference/next/image#priority and set it to true, which disables lazy loading.

**Comment by thanhtutzaw** [comment]
> If the image is above the fold (at the top of your page), you might want to use https://nextjs.org/docs/api-reference/next/image#priority and set it to true, which disables lazy loading.

didn't work and it is single photo delaying ( single pokemon details page)
